docs/automation.md

```diff
@@ -0,0 +1,144 @@
+# Automation
+
+Included in the automation folder are types and example content you can install in your Amplience Dynamic Content and Content Hub instances to get started with Stylitics.
+
+Functionality of the automation includes:
+ - Content Type Schemas
+ - Content Types
+ - Sample Content for demo accounts and demo sku's from Stylitics
+ - Default Icons
+ - Default Cards
+ - Default Visualisations
+
+## Prerequisites for automation
+
+Details for what is required to automate into your Amplience account as well as other assumptions and options.
+
+## Mandatory
+In order for this automation to work, you must have the following:
+- An Amplience Dynamic Content Account.
+- A valid API key (`clientId` & `clientSecret`) for this hub with a minumum of a 'Developer' [role](https://amplience.com/developers/docs/concepts/permissions/roles/)
+- Your Amplience Dynamic Content `hubId` to import
+
+## Other assumptions
+
+These are other assumptions for the automation but can be changes to match your Amplience Dynamic Content account setup.
+
+### Amplience Content Rendering Service
+
+The automation has default visualisations for the Amplience Content Rendering Service. These may not be applicable to you unless you are using in your implementation and you have this available in your Amplience account.
+
+If you _**do**_ have this features enabled and you wish to use, simply
+upload the files found in `examples/crs/` folder and publish.
+
+If you _**do not**_ have this feature enabled and will not use your implementation follow [configuring visualisations](#icons-cards--visualisations)
+
+### Default Repository
+The automation assumes that you have a repository in your Dynamic Content hub with a 'name' of `content`. This is default in most account setups.
+
+If you have a different setup, or would like to apply this to multiple repositories simply do the following before running any automation:
+
+1. Browse to `amplience-automation/automation-files/type`
+2. Open up all of the `*.hbs` files
+3. Find the `repositories` object and ammend the name to the repository name (s) in your account
+4. Save the files and run the automation
+
+## Running the automation
+The files in `amplience-automation/automation-files` are content types and content type schemas that allow you to configure a Stylitics widget using Amplience content items. Also included are example content items that demonstrate how they can be used.
+
+The easiest way to import these is through a script included with the project. Use `npm run import` to obtain a list of arguments, and provide them like `npm run import -- --clientId a --clientSecret b --hubId c` to import to a specific hub.
+
+## Automation options
+
+These additional options are not mandatory but may be useful:
+
+|Option|Alias|Meaning|
+|:----|:----|:----|
+|`--schemaBaseUri`| |determines what base URI is used for the schema IDs being imported. If not provided, this will default to `https://demostore.amplience.com/`.|
```

Comment:
```suggestion
|`--schemaBaseUri`| |determines what base URI is used for the schema IDs being imported. If not provided, this will default to `https://demostore.amplience.com`.|
```

Just need to be accurate here as the URI the user provides can't end with a forward slash.